cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -171,20 +171,12 @@ func (c *VerifyBlobCmd) Exec(ctx context.Context, blobRef string) error {
 			if err != nil {
 				return fmt.Errorf("getting Fulcio intermediates: %w", err)
 			}
-			co.SigVerifier, err = cosign.ValidateAndUnpackCert(cert, co)
```

Comment:
Is this not still needed?

Reply:
Yeah, it happens inside the verification command here: https://github.com/sigstore/cosign/blob/main/pkg/cosign/verify.go#L636

Reply:
I think I'll make it clearer by removing more code and putting it into pkg/